src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -62,24 +67,26 @@ const defaultProps = {
     policies: {},
 };
 
-function SidebarLinksData({isFocused, allReportActions, betas, chatReports, currentReportID, insets, isLoadingApp, onLinkClick, policies, priorityMode}) {
+function SidebarLinksData({isFocused, allReportActions, betas, chatReports, currentReportID, insets, isLoadingReportData, onLinkClick, policies, priorityMode, network}) {
```

Comment:
```suggestion
function SidebarLinksData({isFocused, allReportActions, betas, chatReports, currentReportID, insets, isLoadingApp, onLinkClick, policies, priorityMode, network}) {
```

Looks like a bad merge here

Reply:
ah, you're right. let me fix this.